keyboards/ploopyco/trackball_thumb/rules.mk

```diff
@@ -0,0 +1,31 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Processor frequency
+F_CPU = 8000000
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+POINTING_DEVICE_ENABLE = yes
+POINTING_DEVICE_DRIVER = pmw3360
+
+ENCODER_ENABLE := no
+OPTS_DEF += -DENCODER_ENABLE
```

Comment:
```suggestion
OPT_DEFS += -DENCODER_ENABLE
```
?

Reply:
Because the optical encoder uses the encoder callbacks for controlling the behavior.  To ensure that if anyone does use the `#ifdef ENCODER_ENABLE` preprocessor, that it is handled correctly.